toolkit/scss/forms/_option-select.scss

```diff
@@ -131,3 +131,15 @@
     }
   }
 }
+
+.govuk-option-select {
```

Comment:
Putting this at the bottom of the file means it could be very easily overwritten if we ever copy/pasted in changes from the GOV.UK component. Maybe put it above the comment about the origin of the code in this file?